shell.nix

```diff
@@ -5,10 +5,10 @@
   niv ? sources.niv,
```

Comment:
Is this still needed if `niv` is being pulled from nixpkgs?

Reply:
it is not!  great catch, thanks